backend/app/Controllers/User/NodeStatusController.php

```
@@ -149,7 +150,8 @@ public function getStatus(Request $request): Response
                         $node['daemonListen'],
                         $node['scheme'],
                         $node['daemon_token'],
-                        10 // Short timeout for status checks
+                        10 // Short timeout for status checks,
+                        WingsUrlHelper::isBehindProxy($node)
```

PHPStan (2.2.5)

[error] Syntax error, unexpected T_STRING, expecting ')' on line 154


(phpstan.parse)